boards/arm/v2m_musca/v2m_musca-nonsecure.dtsi

```diff
@@ -0,0 +1,59 @@
+/ {
+	flash0: flash@230400 {
+		/* QSPI flash */
+		reg = <0x00230400 0x1ffc00>;
```

Comment:
This needs to describe the whole part from the non-secure view.  So it'd be <0x00200000 0x800000>.  We need to use partitions to define the application partition.  for non-secure this would be the correct offset to the address which would be 0x000230400.

Reply:
So, @galak, @agross-linaro, should we agree to do this in the Board DTS?